lib/msnodesqlv8/connection-pool.js

```diff
@@ -32,12 +33,20 @@ class ConnectionPool extends BaseConnectionPool {
         const key = p.substr(2, p.length - 3)
 
         switch (key) {
+          case 'driver':
+            // This allows windows users to choose thier driver
+            if (this.config.options.driver && !SUPPORTED_DRIVERS.includes(this.config.options.driver)) {
+              return reject('Unsupported driver: ' + this.config.options.driver)
```

Comment:
1. You must reject with first class Errors (mssql based errors).
2. Rejecting here isn't going to break the loop or the continued execution of the function. It would be better to throw an actual error and catch it outside the loop so that execution of the `_poolCreate` function is halted

Reply:
I'm thinking then the best thing to do is just not check here and use the value provided and then let the driver crash if it's invalid.